pages/about.html

```diff
@@ -0,0 +1,203 @@
+      </nav>
+    </header>
+    <section class="container headline mt-4">
+      <nav aria-label="breadcrumb">
+        <ol class="breadcrumb justify-content-end me-70">
+          <li class="breadcrumb-item">
+            <a href="/" class="text-dark">
+              <i class="fas fa-home"></i>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">
+            About
+          </li>
+        </ol>
+      </nav>
+      <div class="content text-center mt-5">
+        <h1>
+          <span class="d-lg-none h2 d-block">"Hello Future Programmers"</span>
+          Web School
+        </h1>
+        <p class="m-auto">
+          We are a community of teachers and students. We work togther to create creative people who can
+          make awesome websites. So we have a special curriculum to make you professional web developer.
+          We will teach you the basics which are html, css, and js. After that you will learn framework for css
+          which is bootstrap 5. and a framework for js which is reactJS.
+        </p>
+        <div class="contact mt-5">
+          <h2 class="h6">
+            Please contact us if you have any further questions about web school!
+          </h2>
+          <a href="#" class="text-dark text-decoration-underline">
+            webscholl@gmail.com &raquo;
+          </a>
+        </div>
+      </div>
+      <div class="description mb-5 text-center">
+        <h2 class="h3">
+          What Is Web School?
+        </h2>
+        <p class="m-auto mb-5">
+          Are you tired of wasting your time and searching for useless courses and lost a lot of time? 
+          With Web School Academy, you will learn in the right way and with detailed academic explanations 
+          that explain everything in boring and smooth detail with a promise to understand everything 
+          no matter how difficult it is.
+          We exist 5 years ago. There are a lot of students who graduated from our school. And built their career.
+          And some of them have their own companies. Join now its time to make your success
+        </p>
+        <img src="../imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+      </div>
+    </section>
+
+    <section class="past text-center mb-5">
+      <h2 class="title">
+        See the past of web School
+      </h2>
+      <p>
+        Take a look at the last 2 years of our work
+      </p>
+      <div class="last-years d-flex justify-content-evenly align-items-center flex-wrap">  
+        <div class="years p-5 text-white mt-2">
+          <div class="overlay"></div>
+          <span class="h1 fw-bold">
+            2019
+          </span>
+          <p>
+            Web school at 2019 in Egypt
+          </p>
+        </div>
+        <div class="years p-5 text-white mt-2">
+          <div class="overlay"></div>
+          <span class="h1 fw-bold">
+            2020
+          </span>
+          <p>
+            Web school at 2020 in US
```

Comment:
- [ ] The font color must be white and not be part of the overlay, also, check the size of the images.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132545892-881ea362-928a-4d0e-b201-2b788ea9db62.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132545934-e75ad068-4d5a-4140-93fe-9851eecdca10.png)